cdk.context.json

```diff
@@ -0,0 +1,6 @@
+{
+  "hosted-zone:account=301233867300:domainName=imap-mission.com:region=us-west-2": {
```

Comment:
one quick comment that i wanted to make. Is that an aws account number? you may not want it there.

Reply:
Add this file to .gitignore and removed it from the repository with the command git rm --cached cdk.context.json

Reply:
This unfortunately doesn't get it all. You will need to squash the commits down into one to get rid of the old commits that have this number. Right now I can go back and see the old commit where this number is still present, so it was added then removed, but the _history_ still remains.